test/LoudTestTests.h

```diff
@@ -90,6 +100,15 @@ BENCHMARK_THREAD_TEST_GROUP(LoudTestTests, LoudTests)
     for(const Mezzanine::Testing::TestData& SingleResult : Warnifier)
         { TEST_EQUAL(SingleResult.TestName, Mezzanine::Testing::TestResult::Warning, SingleResult.Results) }
 
+    // Performance Tests
+    class NonPerformantLoudTestTests Performifier;
```

Comment:
I don't recall seeing this syntax before, including the previous code review.  I get that I must have, but also, this is f'ing weird.

Reply:
This is not a normal use of the test Class that is generated by the AUTOMATIC_TEST_GROUP invocation above. Because the class names differ from the filename they aren't auto discovered and aren't include in the normal execution and this test suite is manually invoking these to check that results do something other than just pass.

Reply:
This is not a normal use of the test Class that is generated by the AUTOMATIC_TEST_GROUP invocation above. Because the class names differ from the filename they aren't auto discovered and aren't include in the normal execution and this test suite is manually invoking these to check that results do something other than just pass.